example.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
It looks like the `.ipynb` includes a `KeyboardInterrupt` - Is this intentional?

Reply:
Somewhat, I wanted to run all notebook cells so the python version of the notebook would have all cell numbers. However, I didn't want to run the entire stream in the jupyter notebook. Any suggestions on how to overcome this? Maybe a jupyter notebook is not necessary?

Reply:
This isn't relevant anymore as `example.ipynb` will be replaced with a link to new `mitocheck_data`.